syntax = "proto3";

package sift.common.type.v1;

import "google/api/field_behavior.proto";

option go_package = "azimuth/gen/protos/go/sift/common/type/v1;typev1pb";

message ChannelBitFieldElement {
  string name = 1 [(google.api.field_behavior) = REQUIRED];
  // The index of this element's first bit in the logical bit field array.
  int32 index = 2 [(google.api.field_behavior) = REQUIRED];
  uint32 bit_count = 3 [(google.api.field_behavior) = REQUIRED];
}
